lib/order_manager.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'menu_printer'
+require_relative 'order_checker'
+
+class OrderManager
+
+  def initialize(printer = MenuPrinter, order_checker = OrderChecker)
+    @printer = printer
+    @order_checker = order_checker
+    set_menu
+  end
+
+  def see_menu
+    @printer.to_string(@menu)
+  end
+
+  def place_order(order_array)
+    checker = @order_checker.new(@menu, order_array)
```

Comment:
Conventionally, validation of a model is not a separate responsibility, because you are validating state that should be encapsulated - I would expect to generate an Order, and that order would have methods for validating it's own state.